From 1aaa451470d40b51317c2f3d889cfa19bad68747 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 4 Jun 2017 00:37:16 +0200
Subject: [PATCH 2/2] Enforce use of ANGLE in Mapbox GL plugin

* Otherwise the build of qtlocation will fail
  when Qt is configured for 'desktop OpenGL'
* Requires ANGLE libraries present at build
  time and to use the plugin
---
 src/plugins/geoservices/mapboxgl/mapboxgl.pro | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/plugins/geoservices/mapboxgl/mapboxgl.pro b/src/plugins/geoservices/mapboxgl/mapboxgl.pro
index f35f26dc..2ff41608 100644
--- a/src/plugins/geoservices/mapboxgl/mapboxgl.pro
+++ b/src/plugins/geoservices/mapboxgl/mapboxgl.pro
@@ -38,11 +38,10 @@ qtConfig(icu) {
     include(../../../3rdparty/icu_dependency.pri)
 }
 
-# When building for Windows with dynamic OpenGL, this plugin
-# can only run with ANGLE because Mapbox GL requires at least
+# Enforce use of ANGLE because Mapbox GL requires at least
 # OpenGL ES and does not use QOpenGLFunctions for resolving
 # the OpenGL symbols. -lopengl32 only gives OpenGL 1.1.
-win32:qtConfig(dynamicgl) {
+win32:!qtConfig(angle) {
     qtConfig(combined-angle-lib): LIBS_PRIVATE += -l$${LIBQTANGLE_NAME}
     else: LIBS_PRIVATE += -l$${LIBEGL_NAME} -l$${LIBGLESV2_NAME}
 }
-- 
2.13.2

